### PR TITLE
fix(umi): UMI_DIR might be undefined

### PR DESCRIPTION
**Pull request by fi3ework**
##### Checklist

- [ ] `npm test` passes
- [ ] tests are included
- [ ] documentation is changed or added
- [x] commit message follows commit guidelines

##### Description of change

packages/umi build 失败
image

### CODE REVIEW

File: packages/umi/src/plugins/umiAlias.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { IApi } from '@umijs/types';
 
 export default (api: IApi) => {
   api.chainWebpack(memo => {
-    memo.resolve.alias.set('umi', process.env.UMI_DIR);
+    memo.resolve.alias.set('umi', process.env.UMI_DIR ?? '');
```
Comment (ycjcl868) [inline packages/umi/src/plugins/umiAlias.ts]: 这不需要用空值判断，要么是 undefined ，要么是 string

```suggestion
    memo.resolve.alias.set('umi', process.env.UMI_DIR || '');
```

Comment (fi3ework) [reply packages/umi/src/plugins/umiAlias.ts]: 好像在这个场景下 `??` 和 `||` 没有区别，结果都是一样的 🤔

Comment (ycjcl868) [reply packages/umi/src/plugins/umiAlias.ts]: 嗯嗯，解下冲突

Comment (fi3ework) [reply packages/umi/src/plugins/umiAlias.ts]: 不用了，已经在 master 里解决了，close 了

**Pull request closed without merging**